### PR TITLE
blueshield buff

### PR DESCRIPTION
**Pull request by projectkepler-ru**
an unapologetic buff to the blueshield and returning it to it's former glory

## About The Pull Request
This PR add back stuff blueshield had to be on them at all time (excluding their hell laser rifle)

## Changelog

:cl:
balance: added back the energy revolver to the blueshield arsenal and restored it's intended functionality as the founding father intended
imageadd: a new blueshield revolver sprite ported from project kepler
/:cl:

image

### CODE REVIEW

**Comment by ReturnToZender** [comment]
Blueshield does not need a CENTCOM ID, and has a silver ID for a reason >:(

But other than that, I'm a fan!

File: modular_skyrat/modules/blueshield/code/special.dm [hunk 1]
```diff
@@ -1,24 +1,26 @@
 /obj/item/gun/energy/e_gun/revolver //The virgin gun.
-	name = "energy revolver"
-	desc = "An advanced energy revolver with the capacity to shoot both electrodes and lasers."
-	force = 7
-	ammo_type = list(/obj/item/ammo_casing/energy/electrode, /obj/item/ammo_casing/energy/laser/revolver)
+	name = "X-R12 energy revolver"
+	desc = "An experimental model advanced energy weapon with the capacity to shoot both electrodes and lasers, used by many private defense contractors from Romulus to New Moscow, its also rather heavy and would certainly hurt to get pistol whipped by."
+	force = 22
```
Comment (mcmeiler) [inline modular_skyrat/modules/blueshield/code/special.dm:4]: This seems very, *very* powerful. Are you sure?

Comment (ReturnToZender) [reply modular_skyrat/modules/blueshield/code/special.dm:4]: I think the revolver should be comparable to the HoS' laser.

**Comment by SingingSpock** [comment]
I would recommend keeping the silver ID, but giving them access to all basic departmental areas as part of the trim. That means that they can then be given a few additional places, such as the office of the current acting captain, without compromising their ability to enter the main part of engineering if/when the CE goes down.

**Comment by ReturnToZender** [comment]
> I would recommend keeping the silver ID, but giving them access to all basic departmental areas as part of the trim. That means that they can then be given a few additional places, such as the office of the current acting captain, without compromising their ability to enter the main part of engineering if/when the CE goes down.

They have basic departmental access normally, as well as Head offices.

**Comment by SingingSpock** [comment]
> > I would recommend keeping the silver ID, but giving them access to all basic departmental areas as part of the trim. That means that they can then be given a few additional places, such as the office of the current acting captain, without compromising their ability to enter the main part of engineering if/when the CE goes down.
> 
> They have basic departmental access normally, as well as Head offices.

Incorrect. Their access has long been broken. They get ACCESS_RESEARCH (the R&D room) but not ACCESS_SCIENCE (the actual department). The only Head office they have access to is the Captain, none of the others (not even HoP).

File: modular_skyrat/modules/blueshield/code/special.dm (hunk 1 above)
Comment (projectkepler-ru) [reply modular_skyrat/modules/blueshield/code/special.dm:4]: It is comparable and the revolver isn't more powerful

**Comment by mcmeiler** [comment]
> @mcmeiler
I increased it's melee damage as to make it a punishing weapon for anyone who approaches blueshield and as for any other future concerns, this can be addressed by cracking down on people in general, I have been playing blueshield only as an extension of security WHEN the head of security is in immediate danger or when the head of staff is a pacifist( which is often the case in my timezone)

I can get *why* the change was made, but, when an antag shoves over a blueshield and steals their pistol, they suddenly have a melee weapon comparable to an energy sword in power, for no cost. They can't fire it without a mindshield, but, even that can be mitigated with enough prep, or a break-in into the armory (Which is now more feasible given their 22 damage melee weapon).

**Comment by projectkepler-ru** [comment]
@mcmeiler  I'll reduce it a bit :v

File: modular_skyrat/modules/blueshield/code/special.dm (hunk 1 above)
Comment (projectkepler-ru) [inline modular_skyrat/modules/blueshield/code/special.dm:4]:
```suggestion
	force = 16
```

**Pull request merged**